doc/api.yml

```diff
@@ -1493,6 +1743,16 @@ components:
     PageSize:
       type: integer
       description: Cannot exceed `500`
+    PointingCourseType:
+      type: string
+      enum:
+        - MISSING
+        - LATE
+    PointingType:
```

Comment:
PointingType -> MovementType or something like this.

Reply:
checked, change to AttendanceMovementType